apps/notifications/addNotifications.js

```diff
@@ -0,0 +1,47 @@
+const { JsonFile, Executable } = require('@rushstack/node-core-library');
+const inquirer = require('inquirer');
+
+inquirer
+  .prompt([
+    {
+      type: 'input',
+      name: 'message',
+      message: 'Provide the full text of the announcement to be displayed'
+    }
+  ])
+  .then((answers) => {
+    // the branch will be changed to main once pr is in
+    const spawnResult = Executable.spawnSync(
+      'git',
+      [
+        'cat-file',
+        'blob',
+        'refs/remotes/origin/zhas/cli-notification:common/config/notifications/notifications.json'
+      ],
+      {
+        currentWorkingDirectory: process.cwd()
+      }
+    );
+
+    if (spawnResult.status !== 0) {
+      throw new Error(`git cat-file exited with status ${spawnResult.status}: ${spawnResult.stderr}`);
+    }
+
+    const notificationJson = JSON.parse(spawnResult.stdout);
```

Comment:
When modifying the file, we should be reading it from the local filesystem so that two consecutive calls don't result in erasing newly-added notifications.